src/song.cpp

```diff
@@ -160,6 +167,13 @@ void		Song::init()
 		year = "";
 	}
 
+	/* original year from original date */
```

Comment:
Implementing the option would do the trick. I frankly don't think it's worth it at this stage though, so I suggest we leave it at that, and merge as-is.